protocol/cosi.go

```diff
@@ -147,44 +149,38 @@ func (pc *ProtocolCosi) Dispatch() error {
 }
 
 // StartAnnouncement will start a new announcement.
-func (pc *ProtocolCosi) StartAnnouncement() error {
+func (pc *CoSi) StartAnnouncement() error {
```

Comment:
Not really a fan of having nil parameters but here since announcement does not contains nor do anything I think it's ok..